README.md

```diff
@@ -64,12 +64,14 @@ access_token = "YOUR-MATIRX-ACESSTOKEN"
 transparent_color = { r = 0, g = 0, b = 0, alpha = true }
 animation_format = "webp"
 ```
-The `[sticker]` section ist optional and can be left out.
+The `[sticker]` section is optional and can be left out.
 
 `transparent_color` is used as color for semitransparent pixel in `gif`s.
 The field has no effect, if the sticker is not animated or will be converted to `webp` (default).
 `r`,`g`,`b` must been between 0 and 255 inclusive. 
 
+`animation_format`: is used to convert the stickers, you can either choose `webp` (default) or `gif`.
```

Comment:
If I'm being picky, it's not _the_ stickers but just _animated_ stickers. The other will remain png's.

Reply:
you are. but done